app/views/devise/registrations/edit.html.erb

```diff
@@ -38,6 +38,6 @@
 
 <h3>Cancel my account</h3>
 
-<p>Unhappy? <%= button_to "Cancel my account", registration_path(resource_name), data: { confirm: "Are you sure?" }, method: :delete %></p>
+<p>Unhappy? <%= button_to "Cancel my account", registration_path(resource_name), data: { confirm: "Are you sure?", turbo_confirm: "Are you sure?" }, method: :delete %></p>
```

Comment:
It may need to be changed.
If @hotwired/turbo version is lower than 7.2.0-beta.1, confirmation does not work.

```suggestion
<p>Unhappy? <%= button_to "Cancel my account", registration_path(resource_name), data: { confirm: "Are you sure?" }, form: { data: { turbo_confirm: "Are you sure?" } }, method: :delete %></p>
```

https://github.com/hotwired/turbo-rails/issues/302
> I've tried this :
> 
> <%= button_to "Sign out", logout_path, method: :delete, data: { turbo_confirm: "Are you sure?" } %>
>
> but nothing works.

This issue fixed at hotwired/turbo v7.2.0-beta.1
https://github.com/hotwired/turbo/releases/tag/v7.2.0-beta.1
> Fix data-turbo-confirm to work with multiple submitters inside a form by @feliperaul in https://github.com/hotwired/turbo/pull/564

Reply:
Turbo-rails uses Turbo 7.2.0 since [v1.3.0](https://github.com/hotwired/turbo-rails/releases/tag/v1.3.0).

So I think this compatibility issue could be solved by adding this line.

```ruby
gem "turbo-rails", "~> 1.3"
```

Reply:
Thanks for the info, glad to know this is no longer an issue. I guess we can keep it in the `data` attribute then, instead of the `form data`, and make a note about the compatibility in the wiki for example. Any new apps will be generated with the latest turbo so they shouldn't run into any issue.

Reply:
I added a note to the wiki: https://github.com/heartcombo/devise/wiki/How-To:-Upgrade-to-Devise-4.9.0-%5BHotwire-Turbo-integration%5D#data-confirm-vs-data-turbo-confirm (at the bottom of that section):
![Screenshot 2023-02-07 at 11 08 24](https://user-images.githubusercontent.com/26328/217267509-a5aa70cb-2a32-4f93-9eb9-b697b7b07bb0.png)